python/graphstorm/inference/np_infer.py

```diff
@@ -146,7 +146,7 @@ def infer(self, loader, save_embed_path, save_prediction_path=None,
         if save_prediction_path is not None:
             # save_embed_path may be None. In that case, we need to init nid_shuffler
             if nid_shuffler is None:
-                nid_shuffler = NodeIDShuffler(g, node_id_mapping_file, list(preds.keys)) \
```

Comment:
is this a bug? so it's not triggered in the test?

Reply:
This is a bug. It is not triggerred previously. But as I added more tests, this is triggered.